buildSrc/.classpath

```diff
@@ -1,13 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-  <classpathentry kind="src" path="build/generated-src/antlr"/>
-  <classpathentry kind="src" path="src/main/java"/>
-  <classpathentry kind="src" path="src/test/java"/>
-  <classpathentry kind="lib" exported="true" path="../build/libs/ant-1.8.2.jar"/>
-  <classpathentry kind="lib" exported="true" path="../build/libs/antlr-3.1.3.jar"/>
-  <classpathentry kind="lib" exported="true" path="../build/libs/antlr-runtime-3.1.3.jar"/>
-  <classpathentry kind="lib" exported="true" path="../build/libs/stringtemplate-3.2.jar"/>
-  <classpathentry kind="lib" exported="true" path="../build/libs/swt-debug.jar"/>
```

Comment:
It probably does not, the file was added in 2013 as a part of
RT-31216: Ensure NetBeans development in Gradle build works [Eclipse files]

Turns out, eclipse can work with nested projects.  Shows multiple hits when looking for resources, but it does.

I've added a separate project for ColorCube app as an example, but since this PR is about making all directories to build on Eclipse, I'd rather not fix all the apps in this PR.  (I would keep ColorCube as a reference though).

Reply:
Then best to remove the `buildSrc` project files.

I will look at the apps later. I think it's best to create a project per app while we are touching this point. I don't see a reason not to.

Reply:
As far as I can see NetBeans would not affected by this change.

Reply:
I agree with Nir - eclipse project in buildSrc is unnecessary.  Removing.